virttest/qemu_vm.py

```diff
@@ -1543,6 +1558,8 @@ def add_disable_modern(devices, dev, dev_type):
                     if self.last_boot_index > 0:
                         image_boot = False
                     self.last_boot_index += 1
+            image_pci_bus = _get_pci_bus(image_params, "hba")
+            image_params["pci_bus"] = image_pci_bus.get("aobject", "pci.0")
```

Comment:
Btw do you consider the `hba_pci_bus` for `hba` only? Because this way it's also for the actual disks, when talking about `virtio_blk` disk type. If so then it should be probably called `disk_pci_bus` rather than `hba_pci_bus`, because the `hba` is misleading.